src/components/containers/derived.js

```diff
@@ -16,11 +16,7 @@ const AxesFold = connectAxesToLayout(Fold);
 
 const TraceTypeSection = (props, context) => {
   const {fullContainer} = context;
-  if (
-    fullContainer &&
-    fullContainer._fullInput &&
```

Comment:
I don't know why we were checking `_fullInput` here but it was failing in multi-valued contexts

Reply:
yeah _fullInput is where the true trace type for financial traces lie
_fullData.type for an ohlc trace will show scatter, _fullData._fullInput.type, will show ohlc

And all traces are supposed to have a _fullInput key, with type..so that's why I'm looking here
We could also look in Data.type..

But yeah, if you don't look there, it won't properly look for financials

Reply:
ah, right, I keep forgetting to check stuff with financial traces. right now they cause crashes apparently... joy!